### PR TITLE
Initial App Setup For Admin Settings Backend

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+coverage==6.2
+dj-database-url==0.5.0
+Django==3.1.14
+django-cors-headers==3.2.0
+django-picklefield==2.0
+django-request-logging==0.7.1
+django-rest-framework==0.1.0
+djangorestframework==3.11.2
+fyle==0.28.0
+fyle-integrations-platform-connector==1.17.3
```
Comment (Shwetabhk) [inline requirements.txt:10]: why do we need this?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+fyle-accounting-mappings==1.15.3
```
Comment (Shwetabhk) [inline requirements.txt:11]: why do we need this?

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+fyle-rest-auth==1.1.0
+gunicorn==20.0.4
+idna==2.8
+iniconfig==1.1.1
+isodate==0.6.0
+isort==5.10.1
+lazy-object-proxy==1.6.0
+mccabe==0.6.1
+oauthlib==3.2.1
```
Comment (Shwetabhk) [inline requirements.txt:20]: why do we need this?

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,32 @@
+coverage==6.2
```
Comment (Shwetabhk) [inline requirements.txt:1]: Please relook at all the requirements
this requires a lot of cleaning up

File: apps/users/views.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        """
+        Get User Details
+        """
+
+        return Response(
+            data={'Name': 'Fyle Admin Settings'},
+            status=status.HTTP_200_OK
+        )
+
+
```
Comment (Shwetabhk) [inline apps/users/views.py:22]: remove extra line

Comment (Shwetabhk) [reply apps/users/views.py:22]: only 1 extra line should be there not 2

File: apps/users/tests.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (Shwetabhk) [inline apps/users/tests.py:1]: remove this file

File: apps/users/admin.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
```
Comment (Shwetabhk) [inline apps/users/admin.py:2]: remove all admin.py files

**Comment by Shwetabhk** [comment]
Remove `fyle` django app, I don't see any user for it at the moment

File: admin_settings/settings.py [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+WSGI_APPLICATION = 'admin_settings.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/4.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': BASE_DIR / 'db.sqlite3',
```
Comment (Shwetabhk) [inline admin_settings/settings.py:103]: Fix this, shouldn't be sqlite

File: admin_settings/settings.py [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
```
Comment (Shwetabhk) [inline admin_settings/settings.py:17]: this is only used for sqlite, not needed otherwise

File: requirements.txt (hunk 1 above)
Comment (NileshPant1999) [reply requirements.txt:10]: same for calling platform connector was thinking of supporting everything from start but as discussed will remove this as we are removing fyle app

File: requirements.txt (hunk 2 above)
Comment (NileshPant1999) [reply requirements.txt:11]: yes will remove this

**Pull request merged**